addons/web/static/src/scss/form_view.scss

```diff
@@ -816,18 +816,45 @@
 // Overridden style when form view in modal
 .modal .modal-dialog {
     .o_form_view {
+        .o_statusbar_buttons > .btn {
+            /**
+             * Override to prevent the status bar from increasing height when
+             * inside dialog due to the original margin on those buttons. This
+             * prevents the other status buttons, those on the far right of the
+             * bar, from having a gap between their bottom and the bottom border
+             * of the status bar itself.
+             */
+            margin-top: 2px;
+            margin-bottom: 2px;
+        }
+
```

Comment:
Don't see which form view in dialog has this issue.

Reply:
The one from `project.task` create form for example, because it has both left and right "buttons" on the bar.
This is the one that was reported on the issue.

If you didn't have it before the fix, this might be a firefox specific issue.

Reply:
Indeed, this is specific to Firefox.

Reply:
slight difference between chrome and firefox for BS `.btn` on hover (see added border on hover for `btn-primary` on Firefox)

![chrome-vs-firefox-bs-btn-hover](https://user-images.githubusercontent.com/6569390/91960585-5af9d600-ed0a-11ea-93be-1d8e46110f33.gif)